### PR TITLE
feat: Add intro to multimodal live api notebook

### PR DESCRIPTION
**Pull request by gericdong**
Add intro to multimodal live api notebook to demonstrate the some basic examples such as

- text-to-text generation
- text-to-audio generation
- text-to-audio conversation

### CODE REVIEW

File: gemini/multimodal-live-api/intro_multimodal_live_api.ipynb
Comment (holtskinner) [inline gemini/multimodal-live-api/intro_multimodal_live_api.ipynb:1]: Do we need to publish this Notebook with Raw Web Sockets? Are we expecting users to call the API this way or should we direct them to use the SDK?

Comment (gericdong) [reply gemini/multimodal-live-api/intro_multimodal_live_api.ipynb:1]: @holtskinner: Yes, there have already been some asks for direct access (without SDK) examples.

File: gemini/multimodal-live-api/intro_multimodal_live_api.ipynb [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "k9jAArxzClXz"
+      },
+      "source": [
+        "## Use the Multimodal Live API\n",
+        "\n",
+        "Multimodal Live API is a stateful API that uses WebSockets. This section shows some basic examples of how to use Multimodal Live API for text-to-text and text-to-audio generation."
```
Comment (holtskinner) [inline gemini/multimodal-live-api/intro_multimodal_live_api.ipynb:274]:
```suggestion
        "Multimodal Live API is a stateful API that uses [WebSockets](https://en.wikipedia.org/wiki/WebSocket). This section shows some basic examples of how to use Multimodal Live API for text-to-text and text-to-audio generation."
```

File: gemini/multimodal-live-api/intro_multimodal_live_api.ipynb [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+        "You send a text prompt and receive a model response in audio.\n",
+        "\n",
+        "**Notes**\n",
+        "- Multimodal Live API supports the following voices:\n",
+        "  - Puck\n",
+        "  - Charon\n",
+        "  - Kore\n",
+        "  - Fenrir\n",
+        "  - Aoede\n",
+        "- To specify a voice, set the voice_name within the speech_config object, as part of your session configuration.\n"
```
Comment (holtskinner) [inline gemini/multimodal-live-api/intro_multimodal_live_api.ipynb:384]:
```suggestion
        "- To specify a voice, set the `voice_name` within the `speech_config` object, as part of your session configuration.\n"
```

File: gemini/multimodal-live-api/intro_multimodal_live_api_genai_sdk.ipynb [hunk 3]
```diff
@@ -253,12 +253,32 @@
...
       "metadata": {
-        "id": "Q1DE3s_LIUuE"
+        "id": "b51c5ced31f7"
       },
       "source": [
-        "## **Example 1**: Text to text\n",
+        "## Use the Multimodal Live API\n",
         "\n",
-        "You send one text prompt and receive text response."
+        "Multimodal Live API is a stateful API that uses WebSockets. This section shows some basic examples of how to use Multimodal Live API for text-to-text and text-to-audio generation."
```
Comment (holtskinner) [inline gemini/multimodal-live-api/intro_multimodal_live_api_genai_sdk.ipynb:261]:
```suggestion
        "Multimodal Live API is a stateful API that uses [WebSockets](https://en.wikipedia.org/wiki/WebSocket). This section shows some basic examples of how to use Multimodal Live API for text-to-text and text-to-audio generation."
```

File: gemini/multimodal-live-api/intro_multimodal_live_api_genai_sdk.ipynb [hunk 4]
```diff
@@ -295,9 +315,18 @@
...
+        "You send a text prompt and receive a model response in audio.\n",
+        "\n",
+        "**Notes**\n",
+        "- Multimodal Live API supports the following voices:\n",
+        "  - Puck\n",
+        "  - Charon\n",
+        "  - Kore\n",
+        "  - Fenrir\n",
+        "  - Aoede\n",
+        "- To specify a voice, set the voice_name within the speech_config object, as part of your session configuration.\n"
```
Comment (holtskinner) [inline gemini/multimodal-live-api/intro_multimodal_live_api_genai_sdk.ipynb:329]:
```suggestion
        "- To specify a voice, set the `voice_name` within the `speech_config` object, as part of your session configuration.\n"
```

**Review by holtskinner — APPROVED** [review]
Minor suggestions

**Pull request merged**